Libraries/LibWeb/CSS/CountersSet.cpp

```diff
@@ -142,6 +156,21 @@ void resolve_counters(DOM::AbstractElement& element_reference)
     for (auto const& counter : counter_set)
         element_reference.ensure_counters_set().set_a_counter(counter.name, element_reference, *counter.value);
 
+    // Ad-hoc: maybe update initial value for incremented reversed counters.
+    for (auto const& counter : counter_increment) {
+        auto existing_counter = element_reference.ensure_counters_set().last_counter_with_name(counter.name);
+        if (!existing_counter->value.has_value())
+            continue;
+
+        if (!existing_counter->reversed)
+            continue;
+
+        if (existing_counter->is_explicitly_set_reversed_counter)
+            continue; // Counters which are explicitly set do not need to update the initial counter value if incremented.
+
+        existing_counter->originating_element.update_initial_value_for_reversed_counter__after_increment(counter.name, counter.value.value().value());
```

Comment:
This seems to be the only use of this function, and it has a confusing name, can it just be inlined here?